composer.json

```diff
@@ -6,6 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^8.1",
+        "guzzlehttp/guzzle": "^7.5",
```

Comment:
Added `guzzlehttp/guzzle` because Laravel’s HTTP client (`Http::post()` / `Http::fake()`) requires it; without Guzzle the Task 2 job tests can’t run faked HTTP responses. This keeps tests deterministic and avoids real network calls.